tests/multi_ptr/multi_ptr_prefetch_member.h

```diff
@@ -0,0 +1,84 @@
+
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides code for multi_ptr prefetch member
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_MULTI_PTR_PREFETCH_FUNC_H
+#define __SYCLCTS_TESTS_MULTI_PTR_PREFETCH_FUNC_H
+
+#include "../common/common.h"
+
+#include "../common/section_name_builder.h"
+#include "multi_ptr_common.h"
+
+namespace multi_ptr_prefetch_member {
+
+constexpr int expected_val = 42;
+
+/**
+ * @brief Provides functions for verification multi_ptr prefetch member
+ * @tparam T Current data type
+ * @tparam IsDecoratedT sycl::access::decorated enumeration's field
+ */
+template <typename T, typename IsDecoratedT>
+class run_prefetch_test {
+  static constexpr sycl::access::decorated decorated = IsDecoratedT::value;
+  using multi_ptr_t =
+      sycl::multi_ptr<T, sycl::access::address_space::global_space, decorated>;
+
+ public:
+  /**
+   * @param type_name Current data type string representation
+   * @param is_decorated_name Current sycl::access::decorated string
+   *        representation
+   */
+  void operator()(const std::string &type_name,
+                  const std::string &is_decorated_name) {
+    auto queue = sycl_cts::util::get_cts_object::queue();
+    T value = user_def_types::get_init_value_helper<T>(expected_val);
+    SECTION(section_name("Check multi_ptr::prefetch")
```

Comment:
Maybe like this?
```suggestion
    SECTION(section_name("Check multi_ptr::prefetch()")
```
Because now it looks like member type of `multi_ptr`

Reply:
Yes, thanks for the comment :)
I've added `()` to `multi_ptr::prefetch` in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/352/commits/3118349727ee4edee3df3b3745216fb57de7cbb7